dev-docs/show-outstream-video-ads.md

```diff
@@ -20,68 +20,30 @@ Other than using a slightly different ad unit in your Prebid code on the page, y
 
 There should be no changes required on the ad ops side, since the outstream units use the standard Prebid creative, line item targeting setup, etc.
 
-Note that you have the option to show outstream ads with or without an ad server.  For more information, see the instructions below.
-
 * TOC
 {:toc }
 
 ## Prerequisites
 
-+ Demand from a bidder adapter that supports the `"video-outstream"` media type, and returns a renderer for outstream video in its bid response
-+ For more technical information about renderers, see [the pull request adding the 'Renderer' type](https://github.com/prebid/Prebid.js/pull/1082)
-
-## Option 1. With an ad server
++ Inclusion of at least one demand adapter that supports the `"video"` media type
 
-This section will take you through the code you need to write to show outstream video ads using Prebid.js.
+## Step 1: Set up ad units with the video media type and outstream context
 
-### Step 1. Set up outstream video slot sizes
-{: .no_toc}
+Use the `adUnit.mediaTypes` object to set up your ad units with the `video` media type and assign the appropriate context
 
-In your standard Prebid preamble in the header, configure slot sizes to suit your page layout and/or the available demand for those sizes.
-
-{% highlight js %}
-
-var pbjs = pbjs || {};
-
-// ...
-
-var rightSlotSizes = [[ 300, 250 ], [ 300, 600 ], [ 300, 250 ], [ 100, 100 ]];
-
-// ...
-
-{% endhighlight %}
-
-### Step 2. Set up your ad units with the outstream video media type
-{: .no_toc}
-
-Still in the header, set up your ad units with the `video-outstream` media type.
-
-As far as what fields are supported in the `video` object, that will depend on the rendering options supported by your preferred bidder adaptor(s).  For more information, see [Bidders' Params]({{site.github.url}}/dev-docs/bidders.html).
+The fields supported in a given `bid.params.video` object will vary based on the rendering options supported by each bidder.  For more information, see [Bidders' Params]({{site.github.url}}/dev-docs/bidders.html).
 
 {% highlight js %}
 
 var videoAdUnits = [
   {
     code: 'video1',
     sizes: [ 640, 480 ],
-    mediaType: 'video-outstream',
-    bids: [
-      {
-        bidder: 'appnexusAst',
-        params: {
-          placementId: '5768085',
-          video: {
-            skippable: true,
-            playback_method: [ 'auto_play_sound_off' ]
-          }
-        }
+    mediaTypes: {
+      video: {
+        context: "outstream"
```

Comment:
Minor thing but probably good to use single quotes here for consistency with other strings in this example